packages/components/card/src/card.tsx

```diff
@@ -8,6 +8,10 @@ type TCardProps = {
    * Determines the visual effect of the card. A raised card has a box shadow while a flat card has just a border.
    */
   type: 'raised' | 'flat';
+  /**
+   * Determinse the padding that the conent should have from the card borders
+   */
+  padding: 's' | 'm';
```

Comment:
I wouldn't name this `padding` for a number of reasons.
I think something like `insetSpace` might be more appropriate, as it aligns more with the spacing component.

Reply:
Yup. That's what I suggested above too. `scale` or `insetScale`.